services/nutanix-ai/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+displayName: Nutanix AI
+description: NAI core components.
+type: catalog
+allowMultipleInstances: false
+category:
+  - Artificial Intelligence
+certifications:
+  - certified
+  - supported
+  - airgapped
+licensing:
+  - Ultimate
+  - Enterprise
+requiredDependencies:
```

Comment:
this should be dependencies and not requiredDependencies. which did get u this string?

Reply:
https://github.com/mesosphere/kommander-applications/blob/main/services/nkp-insights/metadata.yaml#L9